src/systemverilog/systemverilog.ts

```diff
@@ -354,11 +354,12 @@ export const language = <ILanguage>{
 			// numbers
 			[/\d*\d+[eE]([\-+]?\d+)?(@floatsuffix)/, 'number.float'],
 			[/\d*\.\d+([eE][\-+]?\d+)?(@floatsuffix)/, 'number.float'],
-			[/0[xX][0-9a-fA-F']*[0-9a-fA-F](@integersuffix)/, 'number.hex'],
-			[/0[0-7']*[0-7](@integersuffix)/, 'number.octal'],
-			[/0[bB][0-1']*[0-1](@integersuffix)/, 'number.binary'],
-			[/\d[\d']*\d(@integersuffix)/, 'number'],
-			[/\d(@integersuffix)/, 'number'],
+			[/\d+/, 'number'],
+			[/'[dD][0-9xXzZ]+_?[0-9xXzZ]*/, 'number'],
```

Comment:
Numbers can have multiple underscores so something like this would be better: `/'[dD][0-9xXzZ_]+/`  I think it's fine to allow '_' at the end as well.

Reply:
I thought that we cannot multiple underscores :/ cool then I will change it